plugins/sentry-backend/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
had to add this file to be able to start the backend at all; it didn't used to make a valid dist folder